23 teams (#396)
EuroPython/website: data/pages-content/thankyou.md

---
title: Thank you for EuroPython 2023
subtitle: Thank you
---

# Thank YOU

> The journey to Prague encompassed a full year of exploration, diligent effort, and unwavering dedication. Our exceptional team made it together with warmth, compassion, humour and creativity! A heartfelt appreciation extends to all those who was part of crafting EuroPython 2023 Prague & Remote.

Reviewer: There was a problem with the original. I _think_ this is better.
```suggestion
The journey to Prague encompassed a full year of exploration, diligent effort, and unwavering dedication. Our exceptional team made it together with warmth, compassion, humour and creativity! A heartfelt appreciation extends to all those who took part in crafting EuroPython 2023 Prague & Remote.
```

operations of the EPS, the legal entity behind the EuroPython conference series.
The EPS 2023 Board were responsible for most of the fiscal, legal and venue
related work for the EuroPython 2023 conference, including
venue & supplier selections and contract negotiations.

- Alexander Hendorf
- Anders Hammarquist (Treasurer)
- Artur Czepiel
- Nicolás Demarchi (Vice-Chair)
- Patrick Arminio
- Raquel Dou (Chair)
- Silvia Uberti
- Vaibhav (VB) Srivastav
- Vicky Twomey-Lee

## Code of Conduct Committee

These humans are responsible for receiving & handling Code of Conduct reports and help
the EuroPython community remain a safe and friendly place:

- Naomi Ceder
- Vicky Twomey-Lee
- Cheuk Ting Ho
- Sebastiaan Zeeff
- Silvia Uberti
- Anders Hammarquist

## Communications

As you peruse the (hopefully lucid but always witty) content on our website, blog, newsletter, and social media, it was created by one of the wordsmiths/meme crafters from our Communications team:

- sangarshanan
- Vaibhav Srivastav (VB)
- Diego Russo
- Nicholas Tollervey

## Design

### Conference Designers

> Our logo, website assets, t-shirts, banners, badges, lanyards, and general aesthetics of EuroPython 2023 are created by our designers

Reviewer: ```suggestion
Our logo, website assets, t-shirts, banners, badges, lanyards, and general aesthetics of EuroPython 2023 were created by our designers
```

- [Anastasia Santoro](https://www.linkedin.com/in/anastasia-santoro-ui-ux-designer-620316157/)
- [Orlando Festa](https://www.linkedin.com/in/orlandofesta/)
<!-- - FIXME - ask Patrick if we should put them down below in the Acknowledgement.  -->

### Design Team
And the rest of the assets, stickers and social cards were scrambled together with love by our Design team.
- Patrick Arminio (our front-end wizard)
- Raquel Dou

## Financial Aid team

> If you are one of the 107 people who participated EuroPython 2023 in Prague or remotely, you have surely exchanged many emails with some of our Financial Aid team members. From spreadsheet wizardry, to responding to countless

Reviewer: ```suggestion
If you are one of the 107 people who participated in EuroPython 2023 in Prague or remotely, you have surely exchanged many emails with some of our Financial Aid team members. From spreadsheet wizardry, to responding to countless
```

And the rest of the assets, stickers and social cards were scrambled together with love by our Design team.
- Patrick Arminio (our front-end wizard)
- Raquel Dou

## Financial Aid team

If you are one of the 107 people who participated EuroPython 2023 in Prague or remotely, you have surely exchanged many emails with some of our Financial Aid team members. From spreadsheet wizardry, to responding to countless
emails and processing all the receipts, here are our kind and organised humans behind the Finaid team:

- Sebastiaan Zeff
- Vicky Twomey-Lee
- Cheuk Ting Ho
- Mannie Young
- Manoj Pandey

## Programme Team

The folks managing Call for Proposals (CFP), community voting and the entire
programme selection & scheduling are:

- Artur Czepiel
- Vaibhav (VB) Srivastav
- Cyril Bitterich
- Rodrigo Girão Serrão
- Diego Russo
- Nicholas Tollervey
- Cristián Maureira-Fredes

### [Speaker Mentorship Programme](/mentorship)

- Theofanis Petkos
- Diego Russo
<!-- - FIXME - Ask Fanis if we are missing people. -->

### Programme Reviewers

The talk selection process was supported by our wonderful reviewers:

- Agata Migalska
- Alexander Hess
- Alisa
- Andrii Soldatenko
- Christian Ledermann

> - Cristián Maureira-Fredes
> - Clinton Roy
> - Cosima Meyer
> - Cristián Maureira-Fredes

Reviewer 2: ```suggestion
- Clinton Roy
- Cosima Meyer
- Cristián Maureira-Fredes
```
removing duplication

<img src="/images/cards/volunteers-23.png" />

## EuroPython Society Board

The annually elected [EuroPython Society (EPS) Board](https://www.europython-society.org/eps-members/#eps-board-members) manages the day-to-day
operations of the EPS, the legal entity behind the EuroPython conference series.
The EPS 2023 Board were responsible for most of the fiscal, legal and venue
related work for the EuroPython 2023 conference, including
venue & supplier selections and contract negotiations.

- Alexander Hendorf
- Anders Hammarquist (Treasurer)
- Artur Czepiel
- Nicolás Demarchi (Vice-Chair)
- Patrick Arminio
- Raquel Dou (Chair)
- Silvia Uberti
- Vaibhav (VB) Srivastav
- Vicky Twomey-Lee

## Code of Conduct Committee

These humans are responsible for receiving & handling Code of Conduct reports and help
the EuroPython community remain a safe and friendly place:

- Naomi Ceder
- Vicky Twomey-Lee
- Cheuk Ting Ho
- Sebastiaan Zeeff
- Silvia Uberti
- Anders Hammarquist

## Communications

As you peruse the (hopefully lucid but always witty) content on our website, blog, newsletter, and social media, it was created by one of the wordsmiths/meme crafters from our Communications team:

> - sangarshanan

Reviewer 3: ```suggestion
- Sangarshanan
```

### Design Team
And the rest of the assets, stickers and social cards were scrambled together with love by our Design team.
- Patrick Arminio (our front-end wizard)
- Raquel Dou

## Financial Aid team

If you are one of the 107 people who participated in EuroPython 2023 in Prague or remotely, you have surely exchanged many emails with some of our Financial Aid team members. From spreadsheet wizardry, to responding to countless
emails and processing all the receipts, here are our kind and organised humans behind the Finaid team:

- Sebastiaan Zeff
- Vicky Twomey-Lee
- Cheuk Ting Ho
- Mannie Young
- Manoj Pandey

## Programme Team

The folks managing Call for Proposals (CFP), community voting and the entire
programme selection & scheduling are:

- Artur Czepiel
- Vaibhav (VB) Srivastav
- Cyril Bitterich
- Rodrigo Girão Serrão
- Diego Russo
- Nicholas Tollervey
- Cristián Maureira-Fredes

### [Speaker Mentorship Programme](/mentorship)

- Theofanis Petkos
- Diego Russo
<!-- - FIXME - Ask Fanis if we are missing people. -->

### Programme Reviewers

The talk selection process was supported by our wonderful reviewers:

- Agata Migalska
- Alexander Hess

> - Alisa

Reviewer 3: Just noticed that Alisa is missing the family name. Probably it should be `Alisa Dammer`

Author: added in https://github.com/EuroPython/website/pull/396/commits/4a5272cda88485783a4c716feb278b4c65ed296b :)

- Ernesto Arbitrio
- Gajendra Deshpande
- Jiawei Wang
- Lisa Carpenter
- Manav Mehra
- Maxim Danilov
- Nabanita Roy
- Nicholas Tollervey
- nikkie
- Pooja Gadige
- Pratibha Jagnere
- Radomir Dopieralski
- Raghav Dave
- Richard Kellner
- Roberto Polli
- Sebastiaan Zeeff
- Sebastian Witowski
- Shagun Sodhani
- Siddharth Gupta
- Tiago Montes
- Valerio Maggio
- Vincent Poulailleau
- Vinícius Gubiani Ferreira

## Operations Team

From managing recording & streaming, to liaising with our wonderful remote and
on-site volunteers, the Operations team have all covered. If you ever received a reply from our helpdesk, chances are you interacted with someone from this team :)

- Theofanis Petkos
- Cyril Bitterich
- Nicolás Demarchi
- Dhanshree Arora
- Shekhar Koirala
- Artem Kislovskiy
- Martin Borus
- Daksh P. Jain
- Sebastiaan Zeeff
- Cristián Maureira-Fredes

### EuroPython 2023 Discord Team

The conference discord community structure and the programme notification bot was created by:
- Sebastiaan Zeeff

The registration discord bot was developed and deployed by:
- Artem Kislovskiy
- Sebastiaan Zeeff
- Cheuk Ting Ho
- Cristián Maureira-Fredes
- Kasisnu Singh
- Dhanshree Arora
- Martin Borus

> <!-- - FIXME - Ask Artem if I am missing someone. -->

Reviewer 3: All good, @artcz do you want to be in the list 😀?

Author: I forgot to add Artur. Fixed now here: https://github.com/EuroPython/website/pull/396/commits/d1674762380730a38ef6bed15a3debc8c42aab57

- Petr Viktorin
- Karolina Surma
- Mia Bajić

Our heartfelt thanks to the mentors of Snake Workshop:
- Petra Vidnerová
- Mirek Brabenec
- Karla Fejfarová
- Kuba Urban
- Monika Bláhová
- Roman Diba
- Jakub Červinka
- Radka Kacová

### [Humble Data Workshop](/humble-data)
- Laís Carvalho
- Jodie  Burchell
- Cheuk Ting Ho

And a big shoutout to our Humble Data Workshop mentors:
- Chiin-Rui Tan
- Stefan Behnel
- Alexander Darby
- Penelope Tollervey
- Artem Kislovskiy
<!-- - FIXME - ask Laís if we are missing anyone -->

### [Women in AI Workshop](/wai)
- Nabanita Roy  
- Liliya Akhtyamova
- Bhawna Singh

 And a huge thank you to the Women in AI team for your support through organising the workshop:
 - Dr. Alessandra Sala
 - Begüm Genç McEvoy
 <!-- - FIXME - ask Nabanita if we are missing anyone -->

### [WASM Summit](/wasm)
- Nicholas Tollervey
- Roman Yurchak

### [Beginners' Conference Orientation](/session/beginner-conference-orientation)
- Cheuk Ting Ho
- Martin Borus

### [Mentored Sprints](/mentored-sprint)
- Cheuk Ting Ho

### [AI Game Tournament](/tournament)
- Neil Vaytet

### Social Events Organisers

#### [Thursday Social Event](/social-event)
- Artem Kislovskiy

> - sangarshanan

Reviewer 3: ```suggestion
- Sangarshanan
```

been streaming, recording & editing our talks, and more importantly,
experimenting with us since 2017! Here are the folks who helped us with
streaming and the smooth hybrid experience:
- Matteo Vanni
- Consuelo Cipriani
- Vincenzo Cicanese
- Simone Giorgi
- Domenico Giovane
- Ambra Lorenzetti
- Alessandro Arese Visconti
<!-- - FIXME -->

### Conference Photographer

The official EuroPython conference and social event photos (to be released) were taken by:
- [ Braulio Lara](https://www.brauliolaraph.com/)

### Hazafele - Social Event Band

The social event was made ever so special by the band "[Hazafele](https://www.facebook.com/profile.php?id=100053175145125)"
- Šimon Jakubíček (double bass)
- David Golas (guitar)
- Michal Grossmann (violin)
- Marek Šnorich (accordion)

### Special Mentions

We would like to thank the developers and companies behind the tools we use. The conference wouldn't be possible without:

- [Pretix](https://pretix.eu/about/en/)
  – special thanks to Raphael Michel & Martin Gross
- [Pretalx](https://pretalx.com/p/about/) – special thanks to Tobias Kunze
- [1Password](https://1password.com/)

> - [Zammad](https://zammad.com/en) (self-hosted)

Reviewer 3: I think, (self-hosted) can be omitted

Author: Thank you for the review @Kislovskiy! Addressed in https://github.com/EuroPython/website/pull/396/commits/4a5272cda88485783a4c716feb278b4c65ed296b :blush:

The annually elected [EuroPython Society (EPS) Board](https://www.europython-society.org/eps-members/#eps-board-members) manages the day-to-day
operations of the EPS, the legal entity behind the EuroPython conference series.
The EPS 2023 Board were responsible for most of the fiscal, legal and venue
related work for the EuroPython 2023 conference, including
venue & supplier selections and contract negotiations.

- Alexander Hendorf
- Anders Hammarquist (Treasurer)
- Artur Czepiel
- Nicolás Demarchi (Vice-Chair)
- Patrick Arminio
- Raquel Dou (Chair)
- Silvia Uberti
- Vaibhav (VB) Srivastav
- Vicky Twomey-Lee

## Code of Conduct Committee

These humans are responsible for receiving & handling Code of Conduct reports and help
the EuroPython community remain a safe and friendly place:

- Naomi Ceder
- Vicky Twomey-Lee
- Cheuk Ting Ho
- Sebastiaan Zeeff
- Silvia Uberti
- Anders Hammarquist

## Communications

As you peruse the (hopefully lucid but always witty) content on our website, blog, newsletter, and social media, it was created by one of the wordsmiths/meme crafters from our Communications team:

- Sangarshanan
- Vaibhav Srivastav (VB)
- Diego Russo
- Nicholas Tollervey

> 

Reviewer 4: Add Raquel Dou here.

## Design

### Conference Designers

Our logo, website assets, t-shirts, banners, badges, lanyards, and general aesthetics of EuroPython 2023 were created by our designers
- [Anastasia Santoro](https://www.linkedin.com/in/anastasia-santoro-ui-ux-designer-620316157/)
- [Orlando Festa](https://www.linkedin.com/in/orlandofesta/)
<!-- - FIXME - ask Patrick if we should put them down below in the Acknowledgement.  -->

### Design Team
And the rest of the assets, stickers and social cards were scrambled together with love by our Design team.
- Patrick Arminio (our front-end wizard)
- Raquel Dou

## Financial Aid team

If you are one of the 107 people who participated in EuroPython 2023 in Prague or remotely, you have surely exchanged many emails with some of our Financial Aid team members. From spreadsheet wizardry, to responding to countless
emails and processing all the receipts, here are our kind and organised humans behind the Finaid team:

- Sebastiaan Zeff
- Vicky Twomey-Lee
- Cheuk Ting Ho
- Mannie Young
- Manoj Pandey

> 

Reviewer 4: Add Raquel Dou here.

Author: A bit confused. Where should I add Raquel to?
